models/conversation.go

```diff
@@ -112,6 +112,41 @@ func (db *DB) GetConversation(id int64) (*Conversation, error) {
 	return conversation, nil
 }
 
+// GetUsersConversations returns all conversations of a user
+func (db *DB) GetUsersConversations(userID int64, sort string) ([]Conversation, error) {
+	var queryString strings.Builder
+	fmt.Fprintf(&queryString, "SELECT c.ID, c.Name, c.Description, c.AvatarURL ")
+	fmt.Fprintf(&queryString, "FROM %s AS c JOIN %s AS m ON c.ID = m.ConversationID ", conversationsTable, mappingsTable)
+	fmt.Fprintf(&queryString, "WHERE UserID=? ORDER BY m.ConversationID ")
+	if sort == "desc" {
```

Comment:
I think by default it should be descending because we usually want the conversations from most recent to least recent.